libsql-client/src/cell_value.rs

```diff
@@ -0,0 +1,57 @@
+//! `CellValue` represents libSQL values and types.
+//! Each database row consists of one or more cell values.
+
+/// Value of a single database cell
+#[derive(Clone, Debug)]
+pub enum CellValue {
+    Text(String),
+    Float(f64),
+    Number(i64),
+    Bool(bool),
```

Comment:
If I'm not mistaken, SQLite does not have native boolean type?

Reply:
It does not. It accepts "true" and "false" keywords though, they are just thin syntactic sugar over 0 and 1. I'm pretty sure it was added because this code is modeled after https://github.com/libsql/sqld/tree/main/packages/libsql-client , which also allows mapping to boolean, is that correct @haaawk?

In any case, eventually we should just unify with the http types as I mentioned above, and then Bool will go away on its own